### DISCUSSION TITLE
high contrast theme?

### DISCUSSION THREAD

**Discussion by billysan** [MagistraNocte/Max-8-Default-Dark-Mode#2] [Ideas]
Hey,

Thank you so much for putting this up!

I was wondering maybe we should create a high contrast (having the objects and cords in a bright color like green or something).

Maybe you could help us understand what are the relevant entries in the theme's JSON?

**Comment by MagistraNocte** [comment]
Hi, sorry but I don't have much time at this moment since I'm graduating, and I can't really help since I just found the entries to change with trial and error, you can do the same to find what you want to change, I'll paste my answer here from the other question:
"You can open the maxtheme file with any text editor (I used Textwrangler on mac) inside you can see a list of entries separated by curly brackets, every entry has an ID, a category and possibly a label (these should give you an idea of what that color is used for, but you're gonna have to do a lot of trial and error) and a color code in RGBA format, you just change that code and save. At the end of the file (in the styledefaults section) there is a list of the most common colors used in the max ui, don't forget to experiment with those too. after saving select the theme again in the max preferences to make changes effective."
